server/lib/configuration.js

```diff
@@ -284,6 +284,14 @@ var conf = module.exports = convict({
       default: '/_/csp-violation',
     }
   },
+  marketing_email: {
+    api_url: {
+      doc: 'User facing URL of the Marketing Email Server',
+      format: 'url',
+      default: 'http://127.0.0.1:1114',
```

Comment:
This port is used by one of the profile server's workers. Can we use something like `3031`?